spacewalk/admin/spacewalk-admin.spec

```diff
@@ -36,7 +36,11 @@ BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 Requires:       %{pythonX}
 Requires:       lsof
 Requires:       spacewalk-base
+%if 0%{?suse_version}
 Requires:       sysvinit-tools
+%else
+Requires:	procps-ng
```

Comment:
Formatting seems broken here. Could you please add a few more spaces to align with the other requires?

Reply:
Difference between tab and spaces... I have updated it.